homeassistant/components/roomba/const.py

```diff
@@ -9,5 +9,5 @@
 CONF_BLID = "blid"
 DEFAULT_CERT = "/etc/ssl/certs/ca-certificates.crt"
 DEFAULT_CONTINUOUS = True
-DEFAULT_DELAY = 1
+DEFAULT_DELAY = 100
```

Comment:
Should we not use 30 sec instead?

Reply:
Maybe 30 sec will be almost.

Reply:
Almost? If it okay as well I suggest we use 30 seconds instead

Reply:
Note that it seems you must update the tests as well

Reply:
I am not familiar with HA development and I don't really understand what I have to do. What do you mean by "it seems you must update the tests as well" ?

Reply:
Check the CI test failure logs. You'll see the default delay needs to be updated in the tests as well.